### ISSUE TITLE
Error in ES Modules Resolving After upgrading to v2 pre-release

### ISSUE THREAD

**Issue by tommyasai** [remix-run/remix#7305]
### What version of Remix are you using?

2.0.0-pre.2

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

I executed `npx upgrade-remix pre`
I also tried `npm i @remix-run/{dev,react,serve,node,css-bundle,eslint-config}@pre --save` but the result was the same.

### Expected Behavior

I expect to run without issue

### Actual Behavior

Upon executing `npm run build && npm run start`, I faced this error.

"Error: Directory import 'xxx/build' is not supported resolving ES modules imported from xxx/node_modules/@remix-run/serve/dist/cli.js
Did you mean to import xxx/build/index.js?"

running `npm run dev` doesn't result in any errors and works as expected.

Would appreciate any guidance on how to resolve this. Thank you!

**Comment by pcattori** [comment]
You'll need to specify the full path to the server build directory in your `start` script:

```diff
- "start": "remix-serve build",
+ "start": "remix-serve ./build/index.js",
```

**Comment by pcattori** [comment]
Fixed by #7308

**Comment by tommyasai** [comment]
Thank you, resolved. I appreciate your prompt support!
